great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -521,8 +526,9 @@ def _reconcile_rule_config(
         effective_rule_config: Dict[str, Any]
         if rule_name in existing_rules:
             rule: Rule = existing_rules[rule_name]
-
-            domain_builder_config: dict = rule_config.get("domain_builder", {})
+            domain_builder_config: dict = RuleBasedProfiler._get_domain_builder_configs(
```

Comment:
@Shinnnyshinshin Just by looking at the declared types of `variables` and `rules`, this is not needed.  However, if the declared types are no longer current with the intended usage, then they should be changed.  Thanks.

Reply:
Removed as well :)